### ISSUE TITLE
Additional trailing slash on some routes

### ISSUE THREAD

**Issue by beniwtv** [passbolt/passbolt_api#142]
Hello!

I have noticed that the user listing wasn't loading when I clicked 'users' and I had two 400 errors:

```
https://mydomain.com//groups.json?contain%5Buser%5D=1&order%5B%5D=Group.name+ASC&filter%5Bhas-users%5D=xxxxxxx-27d4-43d9-9ace-xxxxxxxxxxx
https://mydomain.com//users.json?silentLoading=false&order%5B%5D=Profile.last_name+ASC
```

Reason was the double trailing slash, that does not appear on other routes. Digging through the code I found the places where these were added:
```

app/webroot/js/app/model/user.js
app/webroot/js/app/model/group.js
```

That was then combined into "app/webroot/js/dist/bundles/app/passbolt.js" with the wrong URLs.
Removing these double slashes in these files resolved the issue.

**Comment by stripthis** [comment]
HI @beniwtv, thank you for the bug report, we'll add your fix in the next release. Ref. PASSBOLT-2301

**Comment by stripthis** [comment]
Hi @beniwtv this is fixed with the new v1.6.2 release. Thank you for your contribution, feel free to reopen this issue if there is any follow up needed.
